docs/sources/reference/components/otelcol/otelcol.auth.bearer.md

```diff
@@ -34,10 +34,13 @@ otelcol.auth.bearer "LABEL" {
 
 Name     | Type     | Description                                      | Default  | Required
 ---------|----------|--------------------------------------------------|----------|---------
-`token`  | `secret` | Bearer token to use for authenticating requests. |          | yes
+`filename`| `string` | File name containing the authentication token    |          | no
+`token`  | `secret` | Bearer token to use for authenticating requests. |          | no
 `scheme` | `string` | Authentication scheme name.                      | "Bearer" | no
 
-When sending the token, the value of `scheme` is prepended to the `token` value.
+The auth is taken from `token` or `filename` and one of these two value is required. The `filename` is watched for any new changes.
+
+When sending the auth, the value of `scheme` is prepended to the auth value.
```

Comment:
```suggestion
The authentication is taken from `token` or `filename` and one of these two values is required.
The `filename` is watched for any new changes.

When sending the authentication, the value of `scheme` is prepended to the authentication value.
```

We should spell out authentication... unless it's referring to a specific variable/label... then it should be wrapped in backticks.